### PR TITLE
KOGITO-1189 : Kogito Quick Starts Scripts / User Tasks with Security Update

### PR DESCRIPTION
**Pull request by inodeman**
Hi @evacchi @radtriste can you check this PR? It updates the example with the new diagrams. This was pending the merge for metadata. Updated images, diagram and read me files

### CODE REVIEW

File: process-usertasks-with-security-quarkus/README.md [hunk 1]
```diff
@@ -173,3 +193,9 @@ plus the approver who made the first one.
 	}
 }
 ```
+
+You should see a similar message after performing the second line approval after the curl command
+
+```
+{"id":"f498de73-e02d-4829-905e-2f768479a4f1", "approver":"admin","firstLineApproval:true, "secondLineApproval":true,"traveller":{"firstName":"John","lastName":"Doe","email":"jon.doe@example.com","nationality":"American","address":{"street":"main street","city":"Boston","zipCode":"10005","country":"US"}}}
```
Comment (jstastny-cz) [inline process-usertasks-with-security-quarkus/README.md]: That's basically what's above, seems redundant (and conflicting in a few fields).

File: process-usertasks-with-security-quarkus/README.md [hunk 2]
```diff
@@ -13,9 +13,29 @@ This example shows
 * working with user tasks
 * four eye principle with user tasks
 
-
 <p align="center"><img width=75% height=50% src="docs/images/process.png"></p>
 
+* Diagram Properties (top)
```
Comment (jstastny-cz) [inline process-usertasks-with-security-quarkus/README.md:21]: The lists seems visually to be part of the previous one. Plus there's no introduction of what the screenshots show. Please add there, before this list, a few words mentioning:
* the file that is being shown (at least the name, or path within repo)
* what tools it is being shown in (just a note, not necessarily full list)
* possibly also indicate that there's direct connection between the diagram and steps below

File: process-usertasks-with-security-springboot/README.md [hunk 3]
```diff
@@ -13,9 +13,29 @@ This example shows
 * working with user tasks
 * four eye principle with user tasks
 
-
 <p align="center"><img width=75% height=50% src="docs/images/process.png"></p>
 
+* Diagram Properties (top)
```
Comment (jstastny-cz) [inline process-usertasks-with-security-springboot/README.md:21]: the same as in file above.

File: process-usertasks-with-security-springboot/README.md [hunk 4]
```diff
@@ -150,3 +170,9 @@ plus the approver who made the first one.
 	}
 }
 ```
+
```
Comment (jstastny-cz) [inline process-usertasks-with-security-springboot/README.md]: again, seems redundant

**Review by jstastny-cz — CHANGES_REQUESTED** [review]
Change in extensions of diagram files .bpmn2->.bpmn is intended? Is '.bpmn' actually what should be everywhere now?

There should be also matching PR against master (could be cherry-picked after this is merged).

**Review by evacchi — APPROVED** [review]
good for me as soon as @jstastny-cz comments are addressed

**Comment by inodeman** [comment]
> Change in extensions of diagram files .bpmn2->.bpmn is intended? Is '.bpmn' actually what should be everywhere now?
> 
> There should be also matching PR against master (could be cherry-picked after this is merged).

Hi @jstastny-cz, yes as far as I kno .bpmn is the new format, let me know if otherwise

Will cherry-pick on master after Merge

File: process-usertasks-with-security-quarkus/README.md (hunk 1 above)
Comment (inodeman) [reply process-usertasks-with-security-quarkus/README.md]: Hi @jstastny-cz removed

File: process-usertasks-with-security-quarkus/README.md (hunk 2 above)
Comment (inodeman) [reply process-usertasks-with-security-quarkus/README.md:21]: Hi @jstastny-cz added

File: process-usertasks-with-security-springboot/README.md (hunk 3 above)
Comment (inodeman) [reply process-usertasks-with-security-springboot/README.md:21]: Hi @jstastny-cz added

File: process-usertasks-with-security-springboot/README.md (hunk 4 above)
Comment (inodeman) [reply process-usertasks-with-security-springboot/README.md]: Hi @jstastny-cz removed

**Review by jstastny-cz — APPROVED** [review]
LGTM

**Comment by jstastny-cz** [comment]
Changes in readme and images, no need to wait for passing PR check (previous runs passed).

**Pull request merged**